content/terraform-docs-common/redirects.jsonc

```diff
@@ -761,5 +735,596 @@
+    "destination": "/terraform/cdktf/concepts/providers",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cdktf/examples",
+    "destination": "/terraform/cdktf/examples-and-guides/examples",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cdktf/v:version(0\\.12\\.x)/providers",
+    "destination": "/terraform/cdktf/v0.12.x/concepts/providers",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cli/index.html",
+    "destination": "/terraform/cli/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cli/v:version/index.html",
+    "destination": "/terraform/cli/",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/language/v:version(1\\.(?:1|2|3|4|5|6|7|8|9|10|11)\\.x)/state/remove",
+    "destination": "/terraform/cli/:version/commands/state/rm",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/language/modules/testing-experiment",
+    "destination": "/terraform/cli/test",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/language/v:version(1\\.(?:[6-9]|[1-9]\\d)\\.x|[2-9]\\d*\\.\\d+\\.x)/modules/testing-experiment",
+    "destination": "/terraform/cli/:version/test",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/commands/env",
+    "destination": "/terraform/cli/v1.10.x/commands/env",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/v:version(1\\.(?:1[1-9]|[2-9]\\d)\\.x|[2-9]\\d*\\.\\d+\\.x)/commands/env",
+    "destination": "/terraform/cli/:version/commands/env",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/commands/modules",
+    "destination": "/terraform/cli/v1.10.x/commands/modules",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/v:version(1\\.(?:1|2|3|4|5|6|7|8|9)\\.x)/commands/modules",
+    "destination": "/terraform/cli/v1.10.x/commands/modules",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/commands/push",
+    "destination": "/terraform/cli/v1.10.x/commands/push",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/v:version(1\\.(?:1[1-9]|[2-9]\\d)\\.x|[2-9]\\d*\\.\\d+\\.x)/commands/push",
+    "destination": "/terraform/cli/v1.10.x/commands/push",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cli/v:version(1\\.(?:1|2|3|4|5)\\.x)/test",
+    "destination": "/terraform/cli/:version/commands/test",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud",
+    "destination": "/terraform/cloud-docs",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/index.html",
+    "destination": "/terraform/cloud-docs",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/agents/monitoring",
+    "destination": "/terraform/cloud-docs/agents/metrics",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/docs/cloud/agents",
+    "destination": "/terraform/cloud-docs/agents",
+    "permanent": true
+  },
+  {
+    "source": "/terraform/cloud-docs/api-docs/admin/:slug",
```

Comment:
There is no admin API for TFC, so I just boot them to the main page. I figure that's better than a 404, and they can see in the sidebar that Admin doesn't exist.